### ISSUE TITLE
build failure on Debian 11 after PR #123

### ISSUE THREAD

**Issue by SwampRabbit** [ryonakano/reco#125]
hello @ryonakano it seems that after #123 that building for Debian 11 fails with the following:
```
FAILED: com.github.ryonakano.reco.p/obj-x86_64-linux-gnu/Config.c com.github.ryonakano.reco.p/src/Services/DeviceManager.c com.github.ryonakano.reco.p/src/Services/Recorder.c com.github.ryonakano.reco.p/src/Views/CountDownView.c com.github.ryonakano.reco.p/src/Views/WelcomeView.c com.github.ryonakano.reco.p/src/Views/RecordView.c com.github.ryonakano.reco.p/src/Application.c com.github.ryonakano.reco.p/src/MainWindow.c 
valac -C --pkg libhandy-1 --pkg gstreamer-1.0 --pkg gtk+-3.0 --pkg granite --color=always --directory com.github.ryonakano.reco.p --basedir ../ --gresources=../data/reco.gresource.xml Config.vala ../src/Services/DeviceManager.vala ../src/Services/Recorder.vala ../src/Views/CountDownView.vala ../src/Views/WelcomeView.vala ../src/Views/RecordView.vala ../src/Application.vala ../src/MainWindow.vala
../src/MainWindow.vala:95.44-95.81: error: The name `STYLE_CLASS_DEFAULT_DECORATION' does not exist in the context of `Granite' (granite)
        headerbar_style_context.add_class (Granite.STYLE_CLASS_DEFAULT_DECORATION);
                                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Compilation failed: 1 error(s), 0 warning(s)
```
I added the libhandy-1-dev to the build depends, but no luck, by chance does the version of libgranite-dev need bumped now or is it something else?

**Comment by ryonakano** [comment]
Hmm yes, according to [the release note of Granite](https://github.com/elementary/granite/blob/0817d9835a633c55c90571c9bc7e7892c4342ece/data/granite.appdata.xml.in#L49), this const is only available on 6.0.0 or later. But the version of Granite for Debian 11 is 5.5.0, right? So I'll revert this change for compatibility. ☺️

**Comment by SwampRabbit** [comment]
@ryonakano yes Debian 11 still has 5.5.0, Debian Testing has 6.1.1, but I've run into issues backporting it and then conflicts with applications built with granite 5.4.0 and 5.5.0.  Thank you for the quick change, I'll be keeping an eye on granite 6.x.x backporting.... its been a while since I tried so maybe it won't be an issue in the future.
